release-notes/3.2.1.md

```diff
@@ -0,0 +1,5 @@
+# 3.2.1 Release Notes
+
+## Bug fixes
+
+In earlier versions if a device's replication connection got interrupted some documents may never be replicated to that device leaving it in an unknown state. To avoid this it is recommended that everyone upgrade to 3.2.1 or above as soon as possible. [[#5235](https://github.com/medic/medic-webapp/issues/5235)]
```

Comment:
Actually, older versions didn't have this problem, I introduced it in 3.0 (which uses PouchDB 7.4.x) with adding `seq_interval` parameter. The old version of replication didn't forward it as it was not even implemented in CouchDB 1.x. 
So only upgrades and 3.0 installs are in real danger.